src/mermaid-isomorphic.ts

```diff
@@ -3,13 +3,16 @@ import { type BrowserType, chromium, type LaunchOptions, type Page } from 'playw
 
 declare const mermaid: Mermaid
 
-const html = import.meta.resolve('../index.html')
+const resolve: (path: string) => string = 
+  import.meta.resolve || ((path) => new URL(path, import.meta.url).href);
+
+const html = resolve('../index.html')
 const mermaidScript = {
-  url: import.meta.resolve('mermaid/dist/mermaid.js')
+  url: resolve('mermaid/dist/mermaid.js')
```

Comment:
This won’t work. This is a bare import specifier that must resolve into `node_modules`.